components/compliance-service/api/tests/50_scheduler_spec.rb

```diff
@@ -156,8 +156,15 @@ def cleanup
   end
 
   it "creates new jobs as children of existing ones" do
-    sleep 70
-    all_jobs = GRPC jobs, :list, Jobs::Query.new()
+    counter = 0
+    while counter <= 10
+      sleep 10
```

Comment:
I'm wondering if it makes sense to print here.  I'm worried we are hiding a bit of a bug/inefficiency so it might be good to keep an eye on how long this takes.

Reply:
👍  good point, i can add that!